ts/screens/profile/AppearancePreferenceScreen.tsx

```diff
@@ -19,14 +28,35 @@ type ColorModeChoice = "system" | "dark" | "light";
  * @constructor
  */
 const AppearancePreferenceScreen = (): ReactElement => {
+  const store = useIOStore();
+  const dispatch = useIODispatch();
   const { newTypefaceEnabled, setNewTypefaceEnabled } = useIONewTypeface();
 
+  useFocusEffect(() => {
+    void mixpanelTrack(
```

Comment:
Should we wrap this analytics call in an utility function under `ts/screens/profile/analytics`, like the other ones?

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6744/commits/36a950e86073ba9825ba76f043206bc518e207d6